templates/login.liquid

```diff
@@ -3,6 +3,40 @@
   <head>
     {%- auth0:head -%}
     <link rel="shortcut icon" href="https://www.effectivealtruism.org/favicon-16x16.png">
+    {% if application.name contains "Giving" %}
+    <style>
+      /* GWWC COLORS */
+      :root {
+        --page-background-color: white;
+        --primary-color:  #6c0000;
+        --action-primary-color: #6c0000;
+      }
+      /* setting above variables should be enough, but somehow auth0 also decides to add a second css line with the colors hardcoded, so we need to mark these vars as important */
+      body {
+        background: var(--page-background-color) !important;
+      }
+      button[value="default"] {
+        background-color: var(--primary-color) !important;
+      }
+      /* GWWC LOGO: hide img + abuse h1 to show GWWC logo */
+      #prompt-logo-center {
+        display: none;
+      }
+      h1 {
+        text-indent: -9999999px;
+        font-size: 0px !important;
+      }
+      h1:after {
+        content: "";
+        display: block;
+        width: 120px;
+        height: 120px;
+        margin: var(--logo-alignment);
+        background-image: url(https://upload.wikimedia.org/wikipedia/en/3/38/Giving_What_We_Can_text_logo.jpg);
```

Comment:
Arg, it doesn't seem to be on contentful

Reply:
Ok, it seems like contentful is an awkward choice for this. It lets you upload assets, but seems to want them to be published as part of some larger content page, which we don't need. The thing I recommend using for this is Cloudinary. We currently use cloudinary through sharing my account. We should probably upgrade to a team plan. I searched google drive for "gwwc logo", found a PNG in some ancient folder (🙄), and uploaded it to cloudinary. Here you go:
https://res.cloudinary.com/cea/image/upload/v1629107364/GWWC_square_logo.png

Reply:
(Cloudinary has previously only been used for the Forum. I'm not sure what Sam would want to use.)

Reply:
Updated to use cloudinary

Reply:
thanks!